glib-sys/src/lib.rs

```diff
@@ -6069,6 +6078,11 @@ extern "C" {
         notify: GDestroyNotify,
     ) -> c_uint;
     pub fn g_unix_fd_source_new(fd: c_int, condition: GIOCondition) -> *mut GSource;
+    #[cfg(any(feature = "v2_64", feature = "dox"))]
+    pub fn g_unix_get_passwd_entry(
+        user_name: *const c_char,
+        error: *mut *mut GError,
+    ) -> *mut passwd;
```

Comment:
For 'passwd` I added non-unix replacement.

Reply:
Maybe better make this UNIX-only? That's what it has in the name even :)

Reply:
Changed function to unix-only, but still needed replacement for "dox"